osu.Game/Screens/Multi/Play/TimeshiftPlayer.cs

```diff
@@ -64,7 +64,7 @@ private void load()
             {
                 failed = true;
 
-                Logger.Error(e, "Failed to retrieve a score submission token.");
+                Logger.Error(e, "Failed to retrieve a score submission token.\n\nThis may happen if you are not running an official release of osu! (ie. you are self-compiling).");
```

Comment:
Wondering whether we want an android-specific message too as I imagine someone will inevitably roll along and ask why downloading the official APK causes this to pop up.

Reply:
android should work fine. there's already server-side allowance for this (see https://github.com/ppy/osu-web/pull/6449).

Reply:
Oh, right. Missed that as I don't look at the web channel that often.